crud/compare/conditions.lua

```diff
@@ -13,6 +13,11 @@ conditions.operators = {
     LE = '<=',
     GT = '>',
     GE = '>=',
+    NO = '@<>',
+    BTW = '@>><<',
+    OUT = '@<<>>',
+    OUTN = '@<<!>>',
+    INC = '@>)',
 }
 
 local tarantool_iter_by_cond_operators = {
```

Comment:
It seems rather weird that these table isn't updated after adding new operators.